modules/dbfs/filestorage.go

```diff
@@ -181,11 +181,12 @@ func (di *DatabaseImpl) fileCopy(src string, dst string) error {
 	return out.Sync()
 }
 
-func (di *DatabaseImpl) calculateFilePath(relpath string, filename string, projectID int64) (string, error) {
+// cleanPath cleans the relative filepath given and verifies that the filename is safe
+func (di *DatabaseImpl) cleanPath(relativePath string, filename string, projectID int64) (string, error) {
 	if strings.Contains(filename, filePathSeparator) {
 		return "", ErrMaliciousRequest
 	}
-	cleanPath := filepath.Clean(relpath)
+	cleanPath := filepath.Clean(relativePath)
```

Comment:
go convention is to shorten some of these names if it's still clear; relPath is quite explicit without putting the entire word